src/main/groovy/org/pih/warehouse/core/date/DateParser.groovy

```diff
@@ -0,0 +1,55 @@
+package org.pih.warehouse.core.date
+
+import java.time.Instant
+import java.time.LocalDate
+import java.time.ZonedDateTime
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+/**
+ * A simple wrapper component on the individual parsers.
+ * Exists purely for convenience so that we only need to wire in a single component.
+ */
+@Component
+class DateParser {
```

Comment:
This class allows us to invoke all the parsers in a single place. It's just a convenience thing